index.js

```diff
@@ -252,8 +252,9 @@ module.exports = function svelte(options = {}) {
 				const compiled = compile(
 					code,
 					Object.assign(base_options, fixed_options, {
-						name: capitalize(sanitize(id)),
 						filename: id
+					}, major_version >= 3 ? null : {
```

Comment:
Would it be worth extracting the version check into a module-level constant? There's half a dozen times occurrences.

```javascript
const SVELTE3 = major_version >= 3;
// and/or
const LEGACY = major_version < 3;
```